src/globalNotification/globalNotificationContainer.tsx

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
-import { useGlobalNotifications } from './globalNotificationProvider';
+import GlobalNotification from './globalNotification';
+import { Notification } from './types';
+import { connect } from 'react-redux';
+import { RootState } from '../root/rootReducer';
 
-const GlobalNotificationContainer = (): JSX.Element | null => {
-  const { render } = useGlobalNotifications();
-  return <div className="GlobalNotificationContainer">{render()}</div>;
+interface State {
+  notifications: Notification[];
+}
+
+interface Props {
+  notifications: Notification[];
+}
+
+const GlobalNotificationContainer = ({
+  notifications,
+}: Props): JSX.Element | null => {
+  return (
+    <div className="GlobalNotificationContainer">
+      {notifications.map((n) => (
+        <GlobalNotification
+          key={n.id}
```

Comment:
Seems to be copied from the pre-merge fixes form, i.e. can again be written in a shorter form using a splat, which also includes the now omitted label.

Reply:
true.